### PR TITLE
doc: updated READMEs for late install steps.

### PR DESCRIPTION
**Pull request by ricksalsa**
Updated the README with the latest installation steps for the collector.

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -46,14 +46,14 @@ helm search repo aspenmesh
...
-aspenmesh/aspenmesh-collector    0.1.0        	0.1.0       Aspen Mesh Collector
+aspenmesh/aspenmesh-collector   0.1.7        	0.1.0       Aspen OpenTelemetry Collector
 
 ```
 
 ### Next
 
 See the README for each application for configuration and next steps
- * [Aspen Mesh Collector](./docs/aspenmesh-collector.MD)
+ * [Aspen OpenTelemetry Collector](./docs/aspenmesh-collector.MD)
```
Comment (matthieubouchard) [inline README.MD:56]: do you want Aspen _Mesh_ OpenTelemetry?

File: charts/aspenmesh-collector/Chart.yaml [hunk 2]
```diff
@@ -1,9 +1,9 @@
 apiVersion: v2
 name: aspenmesh-collector
 type: application
-version: 0.1.6
+version: 0.1.7
 appVersion: 0.1.0
-description: Aspen Mesh Collector
+description: Aspen OpenTelemetry Collector
```
Comment (matthieubouchard) [inline charts/aspenmesh-collector/Chart.yaml:6]: same question

**Review by matthieubouchard — APPROVED** [review]
LGTM. as long as you've tried that with the --create-namespace flag and it works alright, I can update that in the UI too. My only question was on the naming...leaving out the "mesh" part of Aspenmesh in several places

File: docs/aspenmesh-collector.MD [hunk 3]
```diff
@@ -1,40 +1,52 @@
-# Aspen Mesh Collector
+# Aspen OpenTelemetry Collector
 
-The Aspen Mesh Collector collects telemetry data from Kubernetes and Istio and sends it to the Aspen Mesh Management Site.
+The Aspen OpenTelemetry Collector collects telemetry data from Kubernetes and your Service Mesh and sends it to the [Aspen Mesh Application Intelligence Platform](https://aspenmesh.io/).
```
Comment (matthieubouchard) [inline docs/aspenmesh-collector.MD:3]: same question

File: README.MD (hunk 1 above)
Comment (ricksalsa) [reply README.MD:56]: No, slowing phasing out `Mesh`.

File: charts/aspenmesh-collector/Chart.yaml (hunk 2 above)
Comment (ricksalsa) [reply charts/aspenmesh-collector/Chart.yaml:6]: See above.

File: docs/aspenmesh-collector.MD [hunk 4]
```diff
@@ -1,40 +1,52 @@
...
-cluster: my-cluster
-replica: my-replica
-    
-```
+[Register for a new account](https://console.cloud.aspenmesh.io/v1/oauth2/register) if you haven't signed up for Aspen Mesh.
 
 ### Step 4
 
-Install the Aspen Mesh Collector in an existing cluster. 
+Install the Aspen OpenTelemetry Collector in an existing Kubernetes cluster. You'll need your API key from **Step 3**.
```
Comment (matthieubouchard) [inline docs/aspenmesh-collector.MD:35]: ditto

**Pull request merged**